### PR TITLE
chore: adding link to the doc of how to create a connection

### PR DESCRIPTION
**Pull request by mutianf**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/java-bigtable-hbase/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> ☕️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,87 @@ which makes it easy for development teams to get started.
...
+    import org.apache.hadoop.conf.Configuration;
+    import org.apache.hadoop.hbase.TableName;
+    import org.apache.hadoop.hbase.client.Connection;
+    import org.apache.hadoop.hbase.client.Table;
+    ```
+
+    Then create the connection and connect to the table:
+
+    ```java
+    Configuration configuration = BigtableConfiguration.configure("project-id", "instance-id");
```
Comment (igorbernstein2) [inline README.md:82]: I think you meant Connection connection = BigtableConfiguration.connect(...)

File: README.md [hunk 2]
```diff
@@ -60,6 +60,87 @@ which makes it easy for development teams to get started.
...
+    AsyncConnection connection = ConnectionFactory.createAsyncConnection(configuration).get();
+    AsyncTable table = connection.getTable(TableName.valueOf("table-id"));
+    ```
+
+  * Option 3: Configure connection with `hbase-site.xml` file
+
+    Add the following configuration to your `hbase-site.xml` under your project's `src/main/resources` directory.
+
+    ```xml
```
Comment (igorbernstein2) [inline README.md:108]: I think you need 2 different examples for 1x & 2x

File: README.md [hunk 3]
```diff
@@ -60,6 +60,110 @@ which makes it easy for development teams to get started.
   libraryDependencies += "com.google.cloud.bigtable" % "bigtable-hbase-1.x" % "2.0.0-alpha2"
   ```
 [//]: # ({x-version-update-end})
+
+* Connecting to Cloud Bigtable
```
Comment (kolea2) [inline README.md:64]: We have examples here: https://cloud.google.com/bigtable/docs/hbase-connecting. I think that is the more appropriate place to add more?

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kolea2) [reply README.md:64]: (and instead we can link to the docs via the README)

Comment (mutianf) [reply README.md:64]: Sounds good!

**Review by kolea2 — APPROVED** [review]

**Pull request merged**